viewer/packages/tools/src/Measurement/MeasurementTool.ts

```diff
@@ -337,6 +342,19 @@ export class MeasurementTool extends Cognite3DViewerToolBase {
     return this._measurements.map(measurement => measurement.getMeasurement());
   }
 
+  /**
+   * Hide/unhide all measurements
+   * @param enable
+   */
+  visible(enable: boolean): void {
+    this._measurements.forEach(measurement => {
+      measurement.visible(enable);
+    });
+    this._htmlOverlay.visible(enable);
```

Comment:
Related to previous - do we always want to show labels whenever measurements are toggled on? (I can agree that it makes sense to hide them when the measurement line is hidden, though).